db/migrate/20171003201727_add_expires_minutes_before_to_reservations.rb

```diff
@@ -0,0 +1,5 @@
+class AddExpiresMinutesBeforeToReservations < ActiveRecord::Migration
+  def change
+    add_column :reservations, :expires_mins_before, :integer
```

Comment:
We should validate the numericality of this field and that it's only integers. There's a part of me that wants to allow negative numbers (if the timeinput will allow that), although that might be moot with being allowed to schedule over an admin reservation (I'm thinking that time opens up in the past so you can retroactively reserve for people).

Reply:
Added.